src/main/java/com/spotinst/sdkjava/model/ISpotOceanEcsClusterLaunchSpecRepo.java

```diff
@@ -0,0 +1,6 @@
+package com.spotinst.sdkjava.model;
```

Comment:
This file should go to 'repo' folder.

Reply:
Unable to move this interface as the partent interface access specifier is not public.